src/constants/MarkdownFiles/posts/2025-06-22-gsoc-25-diwangshu-week03.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "GSoC ’25 Week 03 Update by Diwangshu Kakoty"
+excerpt: "AI with Reasoning Capabilities"
+category: "DEVELOPER NEWS"
+date: "2025-06-22"
+slug: "diwangshu-week03"
```

Comment:
A slug similar to the name you have for the file would be more consistent with the naming of other posts: 2025-06-22-gsoc-25-diwangshu-week03.md